README.md

```diff
@@ -1,2 +1,55 @@
-# IPFS-Ethereum-Hackathon
-Repo for hackathon submissions 
+# IPFS Ethereum Hackathon 2017!
+### February 5th to 11th
+
+
+Welcome to the first ever IPFS Ethereum Hackathon! Brought to you by your friendly
+neighborhood MetaMask.
+
+## Schedule and Over view:
+This is a remote Hackathon. You do not need to be in person for the over all hackathon.
+However their is an in person options for some of the events happening in the week (location Seattle)
```

Comment:
However there are in person options for some of the events happening in the week (location Seattle)